elements.html

```diff
@@ -61,6 +61,49 @@ <h3>Preformatted</h3>
 print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
 								</section>
 
+								<!-- another section example-->
```

Comment:
@MandyMeindersma I think this page is benign, but can we remove this page's diff from the PR?

Reply:
This page has a bunch of precoded things that we can copy and paste. Mark and I use it a lot so I would love to leave it here.